src/router/index.js

```diff
@@ -13,11 +13,7 @@ const routes = [
   {
```

Comment:
once are you using dynamic component's import for route, and another time are you import component at the top of file. Keep one convention